google/resource_compute_instance_test.go

```diff
@@ -1187,6 +1248,49 @@ func testAccComputeInstance_attachedDisk(disk, instance string) string {
 	}`, disk, instance)
 }
 
+func testAccComputeInstance_bootDisk(instance string) string {
+	return fmt.Sprintf(`
+	resource "google_compute_instance" "foobar" {
```

Comment:
I'll do it in a separate PR so it's the same for all of them since there are a lot of tests in this file.